mlflow/pytorch/__init__.py

```diff
@@ -571,3 +571,16 @@ def predict(self, data, device="cpu"):
             predicted = pd.DataFrame(preds.numpy())
             predicted.index = data.index
             return predicted
+
+
+def autolog(log_every_n_iter=1):
+    """
+    Wrapper for `mlflow.pytorch._pytorch_autolog.autolog` method.
+    Only PyTorch lightning is supported for this autolog feature.
+
+    :param log_every_n_iter: parameter to log metrics once in `n` iteration. By default, metrics
```

Comment:
Should we rename this to `log_every_n_epoch` for clarity? I know @andrewnitu was in the progress of reworking some of our per-epoch logging parameters, I'll check with him Monday for his recommendation on how this param should be named / behave